dkg/dkg.go

```diff
@@ -609,7 +609,7 @@ func dvsFromShares(shares []share) ([]cluster.DistValidator, error) {
 	return dvs, nil
 }
 
-func forkVersionToNetwork(forkVersion []byte) (string, error) {
```

Comment:
should we move `forkVersionToNetwork` and `networkToForkVersion` functions to a "utils" file?